README.md

```diff
@@ -201,3 +201,49 @@ The license makes no restrictions on how this tool should be used, but keep in
 mind that, as any electronic voting system, it can only be trusted as long as
 the unanonymized vote ballots are made public as soon as the vote closes, which
 may or may not be OK depending on the type of election you are using this for.
+
+## How does the tool pick the winner?
+
+The default vote method is the Condorcet winner. This repository is probably not the best place to
+learn about vote theory, but all you need to know it's the best option to pick an option that
+majority of voters will agree with.
+
+Let's for the sake of simplification take a situation where you are the only voter, and there are
+three candidates (A, B, and C). You set score 100000 to candidate A, score -100000 to candidate B,
+and 0 to candidate C. Here's how Caritat will count the votes:
+
+1. Caritat puts up A against B, see that you set a higher score to A, and count your vote for A.
+   _A gets more vote than B, A wins the duel against B._
+2. Caritat puts up B against C, see that you set a higher score to C, and count your vote for C.
+   _C gets more vote than B, C wins the duel against B._
+3. Caritat puts up A against C, see that you set a higher score to A, and count your vote for A.
+   _A gets more vote than C, A wins the duel against C._
```

Comment:
```suggestion
1. Caritat puts up A against B, see that you set a higher score to A, and count your vote for A.
   Because in this example, you're the only one voter, there's no other ballot to count.
   _A gets 1 vote, B gets 0 votes, A wins the duel against B._
2. Caritat puts up B against C, see that you set a higher score to C, and count your vote for C.
   _C gets 1 vote, B gets 0 votes, C wins the duel against B._
3. Caritat puts up A against C, see that you set a higher score to A, and count your vote for A.
   _A gets 1 vote, C gets 0 votes, A wins the duel against C._
```